examples/mobile/UIComponents/components/view-controls/sub-tab/one.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js"></script>
+	<title>SubTab One</title>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Sample tab</span>
+			</div>
+		</header>
+		<div class="ui-tabs">
+			<div class="ui-sub-tab ui-sub-tab-static">
+				<ul>
+					<li class="ui-li-anchor">
+						<a class="ui-tab-active" href="#">
+							One
+						</a>
+					</li>
+					<li class="ui-li-anchor">
+						<a href="#">
+							Two
+						</a>
+					</li>
+				</ul>
+			</div>
+			<div class="ui-section-changer">
```

Comment:
is this needed here?

Reply:
Yes. The sub-tab co-works with section changer